Deployment of portfolio using github pages

Hadi111jafari opened the pull request:
the portfolio is published using github pages and Added link of the the live project to the README.md file and enhanced.

Hadi111jafari committed:
Update README.md file

Added link of the the live project and enhanced.

ShoiraTa commented on README.md:
```diff
@@ -1,26 +1,42 @@
+## Getting Started
+1. Please go to https://hadi111jafari.github.io/Portfolio/
+2. Click on repositories
+3. Choose repository "Portfolio" or click to  https://hadi111jafari.github.io/Portfolio/
```
- [ ] Props on deploying the application, I only recommend to change the Readme file. The live link is - https://hadi111jafari.github.io/Portfolio/, therefore you cannot use this link in getting started section. Consider using this: 
```
## Live demo 
[Live page](https://github.com/Hadi111jafari/Portfolio)

## Getting Started
1. Please go to https://github.com/Hadi111jafari/Portfolio
2. Click on repositories
3. Choose repository "Portfolio" or click to  https://github.com/Hadi111jafari/Portfolio
4. Then clone the repository to your local machine by using the following steps:
     1. From repository click on code button and copy clone command(Kindly make sure you choose SHH format)
     2. From your terminal type "git clone" command line
     3. On same line paste the the copied clone command from GitHub
     4.  Then click Enter. Now you are the repository/directory on your local machine.
```

Kingstalux replied:
Not resolved

ShoiraTa commented on README.md:
```diff
@@ -1,26 +1,42 @@
+     4.  Then click Enter. Now you are the repository/directory on your local machine.
+     
+### Setup
+GitHub account 
```
- [ ] I recommend deleting this section, because you are already providing setup instructions  in Getting started section

Kingstalux replied:
resolved

ShoiraTa reviewed (changes requested):
Hi @Hadi111jafari ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.giphy.com/media/dEPGknRiEniec/giphy.gif" width="500" height="250" />

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hadi111jafari committed:
Apply requested changes

In README.md file the live link location changed and deleted unnecessary part .

Kingstalux commented on README.md:
```diff
@@ -1,13 +1,15 @@
 
+## Live demo 
+[Live page](https://github.com/Hadi111jafari/Portfolio)
+
```
- [ ] Good job deploying your project but the link provided here is to your GitHub repository and not the live page. Use this link `https://hadi111jafari.github.io/Portfolio/` instead for the live page.

Kingstalux commented on README.md:
```diff
@@ -1,13 +1,15 @@
+[Live page](https://github.com/Hadi111jafari/Portfolio)
+
 ## Getting Started
 1. Please go to https://hadi111jafari.github.io/Portfolio/
```
- [ ] Since you will be using this link on the live demo section, it serves no purpose here. So it will be great if you could remove it in order to make your README more professional.

Kingstalux reviewed (changes requested):
Hi @Hadi111jafari , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966](https://user-images.githubusercontent.com/85113567/156400105-c039088b-b99d-4399-9999-7badf97e4511.gif)




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hadi111jafari committed:
Apply requested changes 

for the second reviewer changes applied on readme file

nicupop729 reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 

Hi @Hadi111jafari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![well-1](https://user-images.githubusercontent.com/64173189/128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
